internal/common/util/lookout.go

```diff
@@ -0,0 +1,15 @@
+package util
```

Comment:
Do you think we should make the name of this file more aligned with the contents?  This file removes nulls from strings.  Or is your plan to have lookout.go be a series of utilities?

Reply:
These are operations which are only relevant for Lookout. One could make an argument to put the `RemoveNullsFromString` function in the `string.go` file as well. Will merge as is, feel free to change it later if it becomes cumbersome.